src/main/webapp/usermap.html

```diff
@@ -55,8 +55,13 @@
             <h1 id="caption">Who is using TEAMMATES?</h1>
             <div id="contentHolder">
                 <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE"></div>
-                <p id="totalCount"></p>
-                <p>Last updated: 30 March 2016</p>
+                    <p id="lastUpdate" class="lastUpdate">Last updated: 14 March 2016</p>
+                 	<h2 class="subcaption align-center">
+                      <span id="totalUserCount" class="totalCount"></span> 
+                      institutions from 
+                      <span id="totalCountryCount" class="totalCount"></span> 
+                      countries
+                   </h2>
```

Comment:
No, it's been updated to 30 Mar 16 in the PR #5163. (and that's what caused the merge conflict)